### ISSUE TITLE
[Research] Canceled Appointments

### ISSUE THREAD

**Issue by kristenmcconnell** [department-of-veterans-affairs/va.gov-team#36855]
# Description
The primary value of the day of check-in appointment page is to show Veterans only the appointments they need to track. The only possible value for the showing canceled appointments is: The VA cancels an appointment and for some reason the Veteran doesn't know.
- Do we have any data on how frequently that occurs?
- What is the current process for when the VA needs to cancel (not the Veteran) --- is it sent via the method preference set by the Veteran e.g. SMS notification, phone call, email etc.
- What is the timeframe that the VA tends to cancel appointments? Do Veterans normally get enough advanced notice that they would know before the day of?

Our initial instinct is to not include canceled appointments on the page, but the answers to the above questions may change our mind.

# Tasks
- [ ] Find answers to the above questions
- [ ] Determine if we need to add canceled appointments to the appointment list in the day of check-in experience.

# Acceptance Criteria
- [ ] If we do need to add it, then create a UX ticket to design the UI. Once the UI is complete, let FE know within #31188.
- [ ] If we don't need to add it, then let FE know within #31188.

**Comment by loripusey** [comment]
@benbrasso-agile6  @kristenmcconnell @niradatta1  Is this research something that can be included in June, July and/or August research trips?

**Comment by acrollet** [comment]
@benbrasso-agile6 @kristenmcconnell @niradatta1 re-pinging on this, we've seen occasional pre-checkin attempts for canceled appointments where the veteran gets a generic error.

**Comment by benbrasso-agile6** [comment]
@acrollet curious, without great detail (or detail, if you feel like it), how are you tracking pre check ins for cancelled appts? 

Although relevant to this ticket, it feels like your data point could be better solved with a better error message. Do you concur? 

Actually, maybe you can answer the first question in this ticket. 

Okay, this isn't in the current sprint, so pausing this for now. However, because I'm working on the July research, I'm actually going to pull this in now. FYI @loripusey

**Comment by benbrasso-agile6** [comment]
To do: incorporate into july convo guide

**Comment by acrollet** [comment]
@benbrasso-agile6 I can't answer the first question with any precision, I just know that sometimes when we get an alert that a pre-checkin failed and I chase down the reason, it turns out to be a canceled appointment. I've seen it a handful of times. (More often, the veteran is trying to complete pre-checkin again for the same appointment - haven't managed to figure out how they can do that)

I do think that better messaging would solve the pre-checkin case and also perhaps help keep veterans from going in for an appointment they didn't know was canceled.

**Comment by loripusey** [comment]
Our plan was to do a message for now but I think this ticket was just to further our understanding of cancelled appointments to decide if we should take further action. I think the message is good for now, but when we get to displaying a list of all future appointments and there is one that was cancelled it might be nice to make sure the Vet knows it was cancelled using visuals.
@benbrasso-agile6 @acrollet

**Comment by benbrasso-agile6** [comment]
@loripusey @kristenmcconnell I've finished considering this for the July trip. I did add a couple questions in the research plan to tackle some of these questions. 

Where do we move this ticket now?

**Comment by loripusey** [comment]
I think we can close it, since the work is not part of the July research tickets
